src/config/config.js

```diff
@@ -14,25 +14,30 @@ if (nodeEnv && nodeEnv.indexOf('production') > -1) {
 }
 
 let appContextPath;
+let layerPath;
 if (appMode === 'start:shogun2') {
   appContextPath = shogun2Path;
+  layerPath = basePath + 'rest/layers';
 }
 if (appMode === 'start:static') {
   appContextPath = staticPath;
+  layerPath = basePath + 'rest/layers';
```

Comment:
static mode does not need a `layerPath` as it does not talk to a backend

Reply:
Fixed in 44c1024.